includes/admin/class-wcmp-export-consignments.php

```diff
@@ -471,4 +471,35 @@ private function setBaseData(): void
             ->setDeliveryType($this->getDeliveryType())
             ->setLabelDescription($this->getFormattedLabelDescription());
     }
+
+    /**
+     * @throws Exception
+     */
+    public function validate(): bool
+    {
+        $this->validateWeight();
+
+        return $this->consignment->validate();
+    }
+
+    /**
+     * @throws \Exception
+     */
+    public function validateWeight(): void
+    {
+        $weight         = $this->getTotalWeight();
+        $maxColloWeight = WCMP_Data::MAX_COLLO_WEIGHT_PER_PACKAGE_TYPE[$this->getPackageType()]
+            ??
+            WCMP_Data::MAX_COLLO_WEIGHT_PER_PACKAGE_DEFAULT;
+
+        if ($weight > $maxColloWeight) {
+            $message = sprintf(
+                __('error_collo_weight_%1$s_but_max_%2$s', 'woocommerce-myparcel'),
+                $weight / 1000,
+                $maxColloWeight / 1000
```

Comment:
Heb je dit ook getest als je het gewicht heb ingesteld op Kilo?

Reply:
Ja, daar had ik ook eerst een comment aan gewijd maar ik begrijp dat er zo min mogelijk comments tussen moeten staan, in ieder geval werkt het dus met alle gewichtsinstellingen, omdat de digitalstampweight in grammen is en de gettotalweight de andere gewichten zelf converteert naar grammen.